src/solvers/flattening/boolbv_quantifier.cpp

```diff
@@ -145,14 +140,14 @@ instantiate_quantifier(const quantifier_exprt &expr, const namespacet &ns)
     return re;
   }
 
-  const exprt min_i = get_quantifier_var_min(var_expr, re);
-  const exprt max_i = get_quantifier_var_max(var_expr, re);
+  const auto min_i = get_quantifier_var_min(var_expr, re);
```

Comment:
use `constant_exprt`, nothing else here makes it obvious we're talking about a constant

Reply:
changed